docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,689 @@
+
+== MicroTx Integration [[MicroTx-Integration]]
+
+- <<MicroTx-Integration-Overview, Overview>>
+- <<MicroTx-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<MicroTx-Integration-Usage, Usage>>
+** <<Building and Deploying the Helidon Teller Application, Building and Deploying the Helidon Teller Application>>
+** <<Building and Deploying the Helidon Participant Application, Building and Deploying the Helidon Participant Application>>
+- <<MicroTx-Integration-References, References>>
+
+
+=== Overview [[MicroTx-Integration-Overview]]
+
+Oracle Transaction Manager for Microservices (MicroTx) Free ensures consistency of XA transactions across Oracle WebLogic Server applications, Helidon microservices, and the JDBC resources deployed in both Helidon and Oracle WebLogic Server.
+
+At this time, the integration with MicroTx Free is offered for evaluation purposes only.
+
+MicroTX ensures consistency of XA transactions using the following distributed applications as an example.
+
+* Helidon Teller application (transaction initiator)
+* Helidon Application (participant application)
+* WebLogic Server (participant application)
+* MicroTx Transaction Coordinator
+
+=== Maven Coordinates [[MicroTx-Integration-Maven-Coordinates]]
+
+Add the following Maven dependency to the `pom.xml` file, as shown in the following example:
+
+[source,xml]
+----
+<dependency>
+     <groupId>com.oracle.tmm.jta</groupId>
+     <artifactId>TmmLib</artifactId>
+     <version>22.3.2</version>
+</dependency>
+----
+
+This dependency is used to refer to the OTMM library that is installed using the following command:
+
+[source,shell] 
+----
+mvn install:install-file -Dfile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-22.3.2.jar -DpomFile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-weblogic-22.3.2.pom
+----
+
+=== Usage [[MicroTx-Integration-Usage]]
+
+The XA transaction is initiated by the Helidon Teller Application, and then distributed to other Helidon and WebLogic Server applications (the participant applications in the transaction).
+
+For information about deploying the MicroTx application WAR file, WebLogic Server JAX-RS web application, and the MicroTx Coordinator Service, see https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/integrating-oracle-weblogic-server-microtx.html#GUID-37A6BA37-5D54-4E25-93C2-4FF5D6E88D30[Integrating WebLogic Server, Helidon, and MicroTx Applications and Services].
+
+==== Building and Deploying the Helidon Teller Application
+
+Download the image for MicroTx Free from https://container-registry.oracle.com/[Oracle Container Registry] and add the `TmmLib` library to the local Maven repository. This library is available in the `<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/` location.
```

Comment:
The instruction is to download the docker image for MicroTx, right? If that's the case, how would the user access `<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/` location? Are they going to this via a docker container? Should they rather download the distribution in https://www.oracle.com/database/transaction-manager-for-microservices/ and extract that in their local path?  BTW, I found this link from a comment by Monica on this same line/topic in review.us.oracle.com.

Reply:
Why is there a mention of `TmmLib` library  and `OTMM` library below? Are they the same? If so, would it be better to use the same terminology?